addons/l10n_it_edi_sdicoop/tests/test_edi_xml.py

```diff
@@ -247,7 +334,6 @@ def test_price_included_taxes(self):
         invoice_etree = self.with_applied_xpath(invoice_etree, "<xpath expr='.//Allegati' position='replace'/>")
         self.assertXmlTreeEqual(invoice_etree, expected_etree)
 
-    @freeze_time('2020-03-24')
```

Comment:
why?

Reply:
Noticed with dako that by freezing time simplified invoice test would not work. By making a bit a research we found that they were unnecessary for all cases.